### ISSUE TITLE
[BUG] <title>

### ISSUE THREAD

**Issue by ivann5637** [microsoft/vscode-dotnettools#1495]
### Describe the Issue

The .NET tools collect usage data in order to help us improve your experience. It is collected by Microsoft and shared with the community. You can opt-out of telemetry by setting the DOTNET_CLI_TELEMETRY_OPTOUT environment variable to '1' or 'true' using your favorite shell.
Read more about .NET CLI Tools telemetry: https://aka.ms/dotnet-cli-telemetry    what to do...?

**Comment by webreidi** [comment]
@ivann5637 yes, setting that environment variable will opt you out of having telemetry collected by the CLI Tools you use. What is the question or issue?

**Comment by Dean-NC** [comment]
Even with this variable set, both `dotnet build` and building inside Visual Studio proper (which I guess uses the CLI) send a lot of requests to the Microsoft IP address 13.107.246.40. I thought disabling CLI telemetry would prevent sending data during builds. Is this a bug?
Occasionally you see issues reported at developercommunity.visualstudio.com about Visual Studio being slow to build. It's likely these people don't know that building the project sends telemetry. If connections to Microsoft servers happens to be slow from your ISP or where you live, it can seem that something's wrong with building.
